javascript/test/js_proto/sensr_proto/global_type_pb.js

```diff
@@ -0,0 +1,409 @@
+// source: sensr_proto/global_type.proto
```

Comment:
Isn't this the generated protobuf files for javascript? If so, can we add them to the gitignore?

Reply:
Will put it in git ignore.